meta/main.yml

```diff
@@ -25,8 +30,9 @@ galaxy_info:
         - 33
     - name: Ubuntu
       versions:
-        - bionic
         - xenial
-  role_name: python
+        - bionic
+        - focal
+  role_name: skeleton
```

Comment:
This `role_name` should not have changed from `python`.

Reply:
Mea culpa.  Reverted in commit 96f1e87.

Thanks for catching this!